### ISSUE TITLE
Make sure logs are rotated for figgy & lae production servers

### ISSUE THREAD

**Issue by escowles** [pulibrary/figgy#515]

**Comment by jrgriffiniii** [comment]
This seems to address https://github.com/pulibrary/princeton_ansible/issues/34 for two specific cases

**Comment by escowles** [comment]
@jrgriffiniii Yes, updating ansible so this is always taken care of by default is the long-term solution.  But I think it's important to make sure this happens before we switch the lae.princeton.edu pointer over to lae-interim, to avoid filling up the disks with logs when they start getting production traffic load.

**Comment by jrgriffiniii** [comment]
This has been addressed within each respective production server environment using the following:

```bash
/opt/lae-blacklight/current/log/*.log {
       weekly
       rotate 10
       copytruncate
       delaycompress
       compress
       notifempty
       missingok
       su root root
}
[...]
/opt/figgy/current/log/*.log {
       weekly
       rotate 10
       copytruncate
       delaycompress
       compress
       notifempty
       missingok
       su root root
}
```

I will proceed with the updates to https://github.com/pulibrary/princeton_ansible

**Comment by jrgriffiniii** [comment]
Apologies, but bandwidth issues during the past two weeks while out of the state compared with additional delays have led me to create a separate issue for this regarding Ansible Playbook support: https://github.com/pulibrary/princeton_ansible/issues/88

**Comment by jrgriffiniii** [comment]
@escowles I'm sorry, I just realized that this may blocked by https://github.com/pulibrary/figgy/issues/545

**Comment by jrgriffiniii** [comment]
This should be fully addressed by https://github.com/pulibrary/princeton_ansible/pull/91
